<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des étudiants</string>
  </property>
  <widget class="QWidget" name="centralWidgetaminee">
   <widget class="QTabWidget" name="tabWidget_2aminee">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_5aminee">
     <attribute name="title">
      <string>Etudiants</string>
     </attribute>
     <widget class="QTabWidget" name="modifieraminee">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>721</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="labelaminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2aminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3aminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idaminee">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomaminee">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenomaminee">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouteraminee">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>290</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5aminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>id classe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idclasseaminee">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idinscriptionaminee">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>260</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6aminee">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id inscription</string>
        </property>
       </widget>
       <widget class="QLabel" name="image_1aminee">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>0</y>
          <width>311</width>
          <height>311</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2aminee">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tabetudiantaminee">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>701</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_idaminee">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par id</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtontri2aminee">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par classe</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_inscriaminee">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par inscri</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4aminee">
       <attribute name="title">
        <string>Rechercher</string>
       </attribute>
       <widget class="QTableView" name="tabetudiant_2aminee">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>701</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_chercheraminee">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>310</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_3aminee">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>310</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3aminee">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_2aminee">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4aminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimeraminee">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7aminee">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_4aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_5aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_6aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_7aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_8aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>id classe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id inscription</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_1aminee">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6aminee">
     <attribute name="title">
      <string>Inscriptions</string>
     </attribute>
     <widget class="QTabWidget" name="modifier_2aminee">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>701</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_8aminee">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_12aminee">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13aminee">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date d'entrée</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14aminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Date de sortie</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_9aminee">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>113</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_2aminee">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_3aminee">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>230</y>
          <width>110</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_4aminee">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>160</y>
          <width>110</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="image_2aminee">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>381</width>
          <height>361</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9aminee">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tabetudiant_3aminee">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>541</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10aminee">
       <attribute name="title">
        <string>Rechercher</string>
       </attribute>
       <widget class="QTableView" name="tabetudiant_4aminee">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>691</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_chercher_2aminee">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_10aminee">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>370</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_13aminee">
       <attribute name="title">
        <string>Informations</string>
       </attribute>
       <widget class="QTableView" name="tabetudiant_5aminee">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>691</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_13aminee">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>370</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_chercher_3aminee">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11aminee">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_11aminee">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17aminee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2aminee">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12aminee">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_12aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date d'entrée</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20aminee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date de sortie</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_2aminee">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEditaminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_2aminee">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_14aminee">
       <attribute name="title">
        <string>Renouvellement</string>
       </attribute>
       <widget class="QTableView" name="tabetudiant_6aminee">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>701</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_14aminee">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>350</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_chercher_4aminee">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>350</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ne pas renouveller</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_5aminee">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>390</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_chercher_5aminee">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>390</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>renouveller</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15aminee">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>350</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16aminee">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>390</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nouvelle date sortie</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBaraminee">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBaraminee">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBaraminee"/>
  <widget class="QToolBar" name="toolBaraminee">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
